x/vesting/keeper/keeper.go

```diff
@@ -58,3 +58,34 @@ func NewKeeper(
 func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
 }
+
+// AddVestingAccount adds the address of vesting account to store.
+// The caller should check the account type to make sure it's a vesting account type.
+func (k Keeper) AddVestingAccount(ctx sdk.Context, addr sdk.AccAddress) {
```

Comment:
this only needs to be called when creating a ClawbackVestingAccount or during Ante Handler check